src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
@kyle I know about it, but this is special case where `CreateArrayElement` will receive as parameter `refract::IElement*` original templatized function is not able to handle this case.